src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/Window/Window.shared.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Linq;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Effects
+{
+	public class Window : RoutingEffect
```

Comment:
Can you rename this class and file adding the suffix `Effect` to follow our pattern?
```suggestion
	public class WindowEffect : RoutingEffect
```